File: Palindrome_Number.java
class Solution {
    public boolean isPalindrome(int x) {
        // if (x<0) return false;
        int rev = 0;
        int a= x;
        while (a >0){            
            rev = (rev * 10) + (a % 10);
            a /=10;
        }
        return x == rev;
       
    }
}

File: 0009-Palindrome Number/Palindrome_Number.cpp
class Solution
{
public:
    bool isPalindrome(int x)
    {
        int a = x;
        long rev = 0;
        while (a > 0)
        {
            rev = (rev * 10) + (a % 10);
            a /= 10;
        }
        return x == rev;
    }
};